types/victory/index.d.ts

```diff
@@ -327,6 +327,112 @@ declare module 'victory' {
         y: number;
     };
 
+    export interface VictoryClipContainerProps {
+        /**
+         * `VictoryClipContainer` renders a single child, or an array of children in group element.
+         */
+        children?: React.ReactElement | React.ReactElement[];
+        /**
+         * The `circleComponent` specifies the element to use when a `VictoryClipContainer` renders
+         * a circular clip path. By default, `VictoryClipContainer` uses the `Circle` component.
+         * @default circleComponent={<Circle/>}
+         */
+        circleComponent?: React.ReactElement;
+        /**
+         * The `className` prop specifies a class name that will be applied to the rendered element.
+         * @example: className="myClipPath"
+         */
+        className?: string;
+        /**
+         * The `clipHeight` prop determines the base height of the rectangular clip path. This prop should be
+         * given as a number. If this prop is not given, it will be calculated based on the height and padding
+         * of the parent chart.
+         */
+        clipHeight?: number;
+        /**
+         * The `clipId` prop may be used to set a deterministic id for the container.
+         * When a `containerId` is not manually set, a unique id will be generated.
+         * It is usually necessary to set deterministic ids for automated testing.
+         */
+        clipId?: number | string;
+        /**
+         * The `clipPadding` prop is used when the clipped area should be larger than the range of a chart.
+         * This prop should be given as an object with `top`, `bottom`, `left`, and `right` properties. Set
+         * the `clipPadding` prop is useful for extending the visible area of a chart in some dimension so
+         * that data or labels are not cut off.
+         */
+        clipPadding?: {
```

Comment:
There's an existing type called `BlockProps` with the same definitions, I think it's worth using it :smile:

Reply:
Agreed.